RecoTracker/SpecialSeedGenerators/src/OutsideInMuonSeeder.cc

```diff
@@ -159,14 +151,10 @@ OutsideInMuonSeeder::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
         if (debug_) LogDebug("OutsideInMuonSeeder") << "\n\n\nSeeding for muon of pt " << mu.pt() << ", eta " << mu.eta() << ", phi " << mu.phi() << std::endl;
         const reco::Track &tk = *mu.outerTrack();
 
-        TrajectoryStateOnSurface state;
-        if (fromVertex_) {
-            FreeTrajectoryState fstate = trajectoryStateTransform::initialFreeState(tk, magfield_.product());
-            dummyPlane_->move(fstate.position() - dummyPlane_->position());
```

Comment:
Potentially creating regression.
In reality (as new code demonstrate) this plane is never used